crates/bevy_window/src/window.rs

```diff
@@ -18,6 +18,7 @@ impl WindowId {
     }
 }
 
+use core::f32;
```

Comment:
I'm not sure what the purpose of this line here is - the constants are also to the primitive types, which are already in the prelude

Reply:
Ups, the Rust plugin for VSCode added that.